lib/runners/processor/golangci_lint.rb

```diff
@@ -0,0 +1,166 @@
+module Runners
+  class Processor::GolangCiLint < Processor
+    include Go
+
+    Schema =
+      StrongJSON.new do
+        let :runner_config,
+            Schema::RunnerConfig.base.update_fields { |fields|
+              fields.merge!(
+                {
+                  target: enum?(string, array(string)),
+                  config: string?,
+                  disable: enum?(string, array(string)),
+                  'disable-all': boolean?,
+                  enable: enum?(string, array(string)),
+                  fast: boolean?,
+                  'no-config': boolean?,
+                  presets: enum?(string, array(string)),
+                  'skip-dirs': enum?(string, array(string)),
+                  'skip-dirs-use-default': boolean?,
+                  'skip-files': enum?(string, array(string)),
+                  tests: boolean?,
+                  'uniq-by-line': boolean?
+                }
+              )
+            }
+
+        let :issue, object(severity: string?)
+      end
+
+    DEFAULT_TARGET = "./...".freeze
+
+    def self.ci_config_section_name
+      # Section name in sideci.yml, Generally it is the name of analyzer tool.
+      "golangci-lint"
+    end
+
+    def analyzer_bin
+      "golangci-lint"
+    end
+
+    def analyzer_name
+      "golangci-lint"
```

Comment:
[suggestion] We're sorry that the implementations are different from each processor (it's "Historical background"), but can you please change to the official name like this? ⬇️ 

```suggestion
      "GolangCI-Lint"
```